components/stripe/stripe.app.js

```diff
@@ -24,14 +28,16 @@ const createOptionsMethod = (collectionOrFn, keysOrFn) => async function ({ prev
     }));
   }
 
-  let nextPageToken = null;
+  let nextPageToken;
```

Comment:
And I think you don't need to set again this variable as `undefined` since the loop will stop once `options` is empry right?

Reply:
I'm not sure, but that makes sense that `async options` would stop being called once the returned `options` array is empty. What would you suggest changing?

Reply:
ok you just need to assign again the `starting_after` with last value and that would do it